package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "deja-vu",
+  "version": "0.0.0",
+  "description": "Filters out duplicitous event receipts as long as we can identify an events original timestamp",
```

Comment:
That was actually kinda the point (was having a bit of fun) since this module arose out of Google pubsub saying one thing in their docs, but us experiencing a different reality.